Examples/Tests/embedded_boundary_python_api/inputs_test_3d_embedded_boundary_picmi.py

```diff
@@ -43,7 +43,7 @@
 
 flag_correct_div = False
 
-solver = picmi.ElectromagneticSolver(grid=grid, method="Yee", cfl=1.0)
+solver = picmi.ElectromagneticSolver(grid=grid, method="ECT", cfl=1.0)
```

Comment:
This test fails when using the "Yee" solver, because the MultiFab that are accessed with PICMI do not exist ; not sure how we want to handle this type of error in general.

Reply:
One way to handle this would be to create a python accessible function that would create those multifabs if they don't exist and return them. But that would still require this test to change since in that case Python would have to take ownership of the multifabs themselves, not just wrappers for them. I think it is okay to not support that at this time. If it turns out there are users that rely on those multifabs we can fix it when they come forward. How does that sound?

Reply:
OK, that works.